src/back/auth/src/service/service.hpp

```diff
@@ -48,13 +49,26 @@ class Service final : public components::LoggableComponentBase {
 	OIDCTokens TokenRefresh(const std::string& refreshToken);
 
 	model::UserInfo GetUserInfo(const std::string& sessionId);
+	model::SessionRefresh RefreshSession(
+		const std::string& sessionId,
+		const std::string& userAgent);
+
+	void SameInactiveSessionSecurityCheck(const std::string& sessionId);
+
+	void DifferentDeviceSecurityCheck(
+		const std::string& currentUserAgent,
+		const std::string& oldUserAgent
+	);	
 
 private:
```

Comment:
Если эти методы нужны только для использования внутри других методов этого же класса, нужно сделать их приватными.
```suggestion

private:
	void sameInactiveSessionSecurityCheck(const std::string& sessionId);
	void differentDeviceSecurityCheck(
		const std::string& currentUserAgent,
		const std::string& oldUserAgent
	);	
```

Reply:
Поправил.